packages/suite/src/views/recovery/index.tsx

```diff
@@ -122,27 +129,48 @@ const Recovery = ({ modal, closeModalApp }: InjectedModalApplicationProps) => {
 
     if (!device || !device.features) {
         return (
-            <Modal
-                size="tiny"
+            <TinyModal
                 heading={<Translation id="TR_RECONNECT_HEADER" />}
                 cancelable
                 onCancel={closeModalApp}
                 data-test="@recovery/no-device"
+                bottomBar={<CloseButton onClick={() => closeModalApp()} variant="TR_CLOSE" />}
             >
                 <Image image="CONNECT_DEVICE" />
-                <Buttons>
-                    <CloseButton onClick={() => closeModalApp()} />
-                </Buttons>
-            </Modal>
+            </TinyModal>
         );
     }
 
     return (
-        <Modal
-            useFixedHeight
+        <StyledModal
             heading={<Translation id="TR_CHECK_RECOVERY_SEED" />}
             totalProgressBarSteps={statesInProgressBar.length}
             currentProgressBarStep={statesInProgressBar.findIndex(s => s === recovery.status) + 1}
+            bottomBar={
```

Comment:
with such complex bottom bars might make sense to put it in a constant first and then pass it as a prop to Modal

Reply:
https://github.com/trezor/trezor-suite/pull/5132/commits/bf313982589ddfc3513cca6b45c6ff2159fb2600